src/main/kotlin/gdx/liftoff/views/dialogs/GenerationPrompt.kt

```diff
@@ -80,12 +82,39 @@ class GenerationPrompt : ViewDialogShower, ProjectLogger, ActionContainer {
       }
 
       intellijPath = process.inputStream.bufferedReader().readLine()
+      intellijIsFlatpak = false
+
       ideaButton.isDisabled = false
     } catch (e: Exception) {
       Tooltip.Builder("Couldn't find IntelliJ in PATH.\nMake sure that you have JetBrains Toolbox and \"Generate shell scripts\" checked in its settings.").target(ideaButton).build()
       ideaButton.isDisabled = true
     }
 //    }
+
+    if (ideaButton.isDisabled && UIUtils.isLinux) {
+      try {
+        val findFlatpakIntellij = arrayListOf("flatpak", "list", "--app", "--columns=application")
+
+        val process = ProcessBuilder(findFlatpakIntellij).start()
+        if (process.waitFor() != 0) {
+          throw Exception("Flatpak Intellij not found")
+        }
+
+        val ids = arrayListOf("com.jetbrains.IntelliJ-IDEA-Ultimate", "com.jetbrains.IntelliJ-IDEA-Community")
+        val output = process.inputStream.bufferedReader().readLines().stream().filter { ids.contains(it) }.findFirst().orElse(null)
+        if (output == null) {
+          throw Exception("Flatpak Intellij not installed")
+        }
+
+        intellijPath = output
+        intellijIsFlatpak = true
+
+        ideaButton.isDisabled = false
+      } catch (e: Exception) {
+        Tooltip.Builder("Couldn't find IntelliJ in PATH or as a flatpak.\nMake sure that you have JetBrains Toolbox and \"Generate shell scripts\" checked in its settings.").target(ideaButton).build()
```

Comment:
Should this message be changed to reflect that you can install IDEA using a way other than JetBrains Toollbox?